### DISCUSSION TITLE
Make data model browsing quicker in collections details view

### DISCUSSION THREAD

**Discussion by magomimmo** [directus/directus#3511] [Feature Requests]
When you're viewing the details of a collection model, currently, you can step back to the data model view by clicking the left arrow icon. 

If you want to visit the previous or the next collection model, which frequently happens, you need to go back to the data model and click again a collection to see its model.

By adding a backward arrow (if the currently selected collection model is not the first) and an forward arrow (if the currently selected collection is not the last) you could browse your data model more quickly

**Comment by benhaynes** [comment]
Just to be clear, you're talking about the Item Detail view (form), or the Collection Detail view (listing)? Also, when you say data-model, I think of the Settings page... are you talking about the Settings (Admin) or the normal user pages?

**Reply by magomimmo** [reply]
Hi Ben sorry to have been unclear:

Settings / Data Model / any collection

From there I'd like to go to the next one or to the previous one (if there are) without going back first to the data model list of available collections.

That said, reasoning about your comment on being more specific, I think that the same style of browsing (i.e. using forward and backward arrow icons) could be very useful even when you want to browse collection records instead of collection models.

**Reply by benhaynes** [reply]
Ahh — that actually seems a little less useful to me, but that's preference. I think that having "prev/next" buttons on item details might be worth looking into... not sure about at the collection level though. Let's see what the community thinks in terms of 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
